### ISSUE TITLE
HelloTriangle on Windows 10 not working

### ISSUE THREAD

**Issue by bolner** [luca-piccioni/OpenGL.Net#144]
When I ran this project with `dotnet run` (left only one csproj file there):
- https://github.com/luca-piccioni/OpenGL.Net/tree/master/Samples/HelloTriangle.NETCore

then I got a window with a blank black background.
I tried the framework versions 2.2 and 2.0.

I didn't get any error messages.

OpenGL.Net version: current master branch
OS: current Windows 10 with all updates
GPU: NVIDIA GeForce GTX 780 Ti

**Comment by PhilT** [comment]
yeah I'm seeing this with .NET Core 3 I created by copying the Core 2.2 sample. Did you get any further @bolner ?

**Comment by bolner** [comment]
@PhilT I didn't. I think that these libraries which claim to support all .NET frameworks on multiple platforms are too costly to maintain, therefore they never work properly. I'll wait until someone creates a ".NET Core"-only library for OpenGL for Windows and Linux. As I think that in the future everyone should switch to .NET Core.

**Comment by PhilT** [comment]
@bolner I've actually just discovered [Silk.NET](https://github.com/Ultz/Silk.NET). It's at 1.0.0 preview 2 with preview 3 coming out later this month. It seems to be quite usable, as in, I managed to checkout the latest code and run the samples. Some guys from OpenTK are working on it. They have an active discord chat. Seems like it could be a good option.
